### PR TITLE
Document support for viewing live stream via live input ID

### PR DESCRIPTION
**Pull request by bkrebsbach**
Document support for static manifest URLs via live input ID, and document
viewing information and relevant APIs.

### CODE REVIEW

File: products/stream/src/content/stream-live/watch-live-stream.md [hunk 1]
```diff
@@ -96,6 +98,43 @@ The response will contain the HLS/DASH URL that can be used to play the current
 }
 ```
 
+## View by live input uid
+
+By using the live input uid in place of a video id in the hls/dash manifest URL, you get a static URL that will return the manifest of the active livestream, or a 204 status code if there is no active live stream.
```
Comment (renandincer) [inline products/stream/src/content/stream-live/watch-live-stream.md]: might want to clarify that the manifest will be 200 while 204 wont' have any content in it.

File: products/stream/src/content/stream-live/watch-live-stream.md [hunk 2]
```diff
@@ -96,6 +98,43 @@ The response will contain the HLS/DASH URL that can be used to play the current
...
+{
+    "isInput": true,
+    "videoUID": null,
+    "live": false
+}
+```
+
+### Live input level viewing options
+
+When viewing a livestream via the live input id, the `requireSignedURLs` and `allowedOrigins` options in the live input recording settings are used. These settings take precedence over any video-level settings.
```
Comment (renandincer) [inline products/stream/src/content/stream-live/watch-live-stream.md]: `These settings take precedence over any video-level settings.` does this imply that if live input settings are empty; video settings are used instead? or are they separate objects with independent configuration

Comment (bkrebsbach) [reply products/stream/src/content/stream-live/watch-live-stream.md]: They are separate objects with independent configuration.

Comment (bkrebsbach) [reply products/stream/src/content/stream-live/watch-live-stream.md]: The input settings are used as defaults for the video, but once a video is created, they can be changed independent of each other.

**Comment by phuho2020** [comment]
Vào 02:24, T.6, 19 Th11, 2021 Renan Dincer ***@***.***> đã
viết:

File: products/stream/src/content/stream-live/watch-live-stream.md (hunk 1 above)
Comment (bkrebsbach) [reply products/stream/src/content/stream-live/watch-live-stream.md]: I've updated this in a force-push, let me know what you think of the new wording.

File: products/stream/src/content/stream-live/watch-live-stream.md (hunk 2 above)
Comment (bkrebsbach) [reply products/stream/src/content/stream-live/watch-live-stream.md]: I've tried to clarify this in a force-push.

File: products/stream/src/content/stream-live/watch-live-stream.md [hunk 3]
```diff
@@ -96,6 +98,43 @@ The response will contain the HLS/DASH URL that can be used to play the current
...
+{
+    "isInput": true,
+    "videoUID": null,
+    "live": false
+}
+```
+
+### Live input level viewing options
+
+When viewing a livestream via the live input id, the `requireSignedURLs` and `allowedOrigins` options in the live input recording settings are used. These settings take are independent of the video-level settings.
```
Comment (dcpena) [inline products/stream/src/content/stream-live/watch-live-stream.md]:
```suggestion
When viewing a livestream via the live input id, the `requireSignedURLs` and `allowedOrigins` options in the live input recording settings are used. These settings are independent of the video-level settings.
```

**Review by dcpena — APPROVED** [review]
Looks good! Left one suggestion to fix a typo but overall 👍

File: products/stream/src/content/stream-live/watch-live-stream.md (hunk 3 above)
Comment (bkrebsbach) [reply products/stream/src/content/stream-live/watch-live-stream.md]: Fixed!

**Review by renandincer — APPROVED** [review]

**Pull request merged**